src/olympia/constants/activity.py

```diff
@@ -405,9 +407,21 @@ class USER_DELETED(_LOG):
     format = _('Account {user} deleted.')
 
 
-class USER_AUTO_DELETED(_LOG):
+class _USER_AUTO_DELETED_METACLASS(type):
+    """Using metaclass is a workaround because we need a class property,
+    (and chaining @classmethod and @property is deprecated)"""
+
+    @property
+    def format(cls):
+        return (
+            _('Account {user} deleted, from Mozilla accounts event.')
+            if amo.is_mza_branding()
+            else _('Account {user} deleted, from Firefox Accounts event.')
```

Comment:
Given what it is about, this will never be exposed to end-users (we can add `admin_event = True` to be certain), it's purely for admins.

Which means it shouldn't need to be translated, and we could word it in a different way to not have to deal with branding issues, avoiding the need for the metaclass workaround.

Reply:
https://github.com/mozilla/addons-server/pull/21202/commits/994e5b3000f9a8f13049d76327c7882aa2e93776